### DISCUSSION TITLE
Adding single tabs to an existing saved session

### DISCUSSION THREAD

**Discussion by Chianti** [sienori/Tab-Session-Manager#788] [Q&A]
Sorry, I can't see this information anywhere...

I have a session already saved on the subject of 'researching a new computer'. That session is closed.

In the current Window/session I have come across a page that I would like to add to that research session.

What is the correct workflow/method for this?

I guess ideally, I would like to be able to open TSM, see the 'current session' listed, and drag/drop the page onto existing saved sessions. It would also be great to be able to drag/drop individual tabs from one saved session to another.

But without this functionality of being able to move tabs around, I'm guessing the workflow is to open the saved session in a new window, move the desired tab to the new window, right click the saved session, and choose 'replace with current window'?

Thanks.

**Comment by adamwinn** [comment]
I came here looking for the exact same thing

**Reply by Chianti** [reply]
I can't see any other tab/window manager which works with pinned and grouped tabs. Also the auto-saving is awesome. Using Google Drive as the cloud sync is brilliant too.

Due to the lack of window management though, I think I'm going to use Tab Session Manager for creating a set of 'default' Windows. Eg. A Google App window, a home recording window, a social media window, a news window, a books/film window, etc. Standard sets of tabs I want to start new sessions with, that aren't going to change very much.

But for managing quite changeable windows of things like current research and tutorials, I'm going to use Cluster Tab Manager (which is a Chrome extension). It's strength is moving tabs between different window sets. I think using both TSM and CTM together is the best of both worlds.

**Comment by sienori** [comment]
I will implement this feature in the future.

**Reply by gskoczylas** [reply]
It would be very useful if it were possible to save not only the current tab, but also all selected tabs.

**Comment by kissingfrogs2003** [comment]
Yes love the idea of a drag and drop function for moving whole windows/groups of tabs into a separate session!
